website/blog/2022-11-30-databend-weekly-70.md

```diff
@@ -0,0 +1,185 @@
+
+## What's Changed
+
+Below is a list of some major changes that we don't want you to miss.
+
+### Exciting New Features :sparkles:
+
+**format**
+
+- better checking of format options ([#8981](https://github.com/datafuselabs/databend/pull/8981))
+- add basic schema infer for parquet ([#9043](https://github.com/datafuselabs/databend/pull/9043))
+
+**query**
+
+- QualifiedName support 'db.table.*' and 'table.*' ([#8965](https://github.com/datafuselabs/databend/pull/8965))
+- support bulk insert without exprssion ([#8966](https://github.com/datafuselabs/databend/pull/8966))
+
+**storage**
+
+- add cache layer for fuse engine ([#8830](https://github.com/datafuselabs/databend/pull/8830))
+- add system table system.memory_statistics ([#8945](https://github.com/datafuselabs/databend/pull/8945))
+- add optimize statistic ddl support ([#8891](https://github.com/datafuselabs/databend/pull/8891))
+
+### Code Refactor :tada:
+
+**base**
+
+- remove common macros ([#8936](https://github.com/datafuselabs/databend/pull/8936))
+
+**format**
+
+- TypeDeserializer get rid of FormatSetting ([#8950](https://github.com/datafuselabs/databend/pull/8950))
+
+**planner**
+
+- refactor extract or predicate ([#8951](https://github.com/datafuselabs/databend/pull/8951))
+
+**processors**
+
+- optimize join by merging build data block ([#8961](https://github.com/datafuselabs/databend/pull/8961))
+
+**new expression**
+
+- allow sparse column id in chunk, redo [#8789](https://github.com/datafuselabs/databend/pull/8789) with a new approach. ([#9008](https://github.com/datafuselabs/databend/pull/9008))
+
+### Documentation :notebook_with_decorative_cover:
+
+- i18n support with crowdin ([#8987](https://github.com/datafuselabs/databend/pull/8987), [#8997](https://github.com/datafuselabs/databend/pull/8997), etc.)
+
+### Thoughtful Bug Fix :wrench:
+
+**base**
+
+- try fix lost tracker ([#8932](https://github.com/datafuselabs/databend/pull/8932))
+
+**meta**
+
+- fix share db bug, create DatabaseIdToName if need ([#9006](https://github.com/datafuselabs/databend/pull/9006))
+
+**mysql handler**
+
+- fix mysql conns leak ([#8894](https://github.com/datafuselabs/databend/pull/8894))
+
+**processors**
+
+- try fix update list memory leak ([#9023](https://github.com/datafuselabs/databend/pull/9023))
+
+**storage**
+
+- read and write block in parallel when compact ([#8921](https://github.com/datafuselabs/databend/pull/8921))
+
+## News
+
+Let's take a look at what's new at Datafuse Labs & Databend each week.
+
+#### Preview of Infer Schema
+
+To load data from a stage or location, users need to create table first. However, sometimes users don't know the file schema, or the schema is too complex / too simple to be input by users.
+
+Allowing infer schema from existing files makes our users' lives easier. Also, this feature will unlock the databend from implementing `select * from @my_stage`.
```

Comment:
Introducing the capability to infer schema from an existing file will make the work much easier. You will even be able to query data from a stage using a SELECT statement like "select * from @my_stage".